www/docs/client/introduction.md

```diff
@@ -0,0 +1,21 @@
+# tRPC Client
```

Comment:
Aren't we missing the headers etc here? Maybe it's not needed

Reply:
Was thinking the same, react docs have it. Will do